docs/README-agent-install.md

```diff
@@ -24,18 +24,21 @@ This document describes Puppet agent installation and setup on Cisco Nexus switc
 #### Platform and Software Minimum Requirements
 
 * The Cisco NX-OS Puppet implementation requires open source Puppet version 4.0 or Puppet Enterprise 2015.2
+* The Cisco IOS XR Puppet implementation requires open source Puppet version 4.3.2 or Puppet Enterprise 2015.3.2
```

Comment:
It does.. but our `metadata.json` doesn't handle any platform differences in version requirements and in fact doesn't mention XR at all.  If XR has the minimum requirements of Puppet 4.3.2 and PE 2015.3.2, then our `metadata.json` should reflect that.  This isn't a necessary change for your PR, but it's something I wanted to clarify.